### DISCUSSION TITLE
Third-Party Dispatch option SDongleA-05

### DISCUSSION THREAD

**Discussion by RikBast** [wlcrs/huawei_solar#823] [General]
After updating my Dongle I now can choose as operation mode also 'Third-party dispatch', however enabling this will make the function forcible charge unavailable. V200xxxSPC118 firmware
<img width="1372" alt="Scherm­afbeelding 2024-10-12 om 13 12 29" src="https://github.com/user-attachments/assets/e09fb149-054a-46f7-93ec-51be2297bc05">

Current setting: Max self-consumption, works.

Just wondering if Third-party dispatch will be an option for the HA Huawei integration in the future and if it will have any advantages against max self-consumption and TOU.

**Comment by RikBast** [comment]
Well, I came to learn that when the command huawei_solar.stop_forcible_charge is issued, the setting max-self-consumption takes over, so your battery will discharge anyway. Exploring Third-party dispatch implementation would be welcome

**Comment by imehun** [comment]
For third party working mode you need write your logic in HA for everything.
